src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -274,7 +274,7 @@ private IEnumerable<string> RecursivelyEnumerateFiles(string folderPath)
                 {
                     foundFiles =
                         foundFiles.Concat(
-                            Directory.EnumerateFiles(
+                            Directory.GetFiles(
```

Comment:
While you're at it, can you add the check for PathTooLongException below using a catch exception filter:
```
catch (Exception ex) when {ex is UnauthorizedException || ex is PathTooLongException) { ... }
```
You might want to check DirectoryNotFoundException in case someone deletes a dir while we're searching.